src/UnderlineNav2/UnderlineNavItem.tsx

```diff
@@ -70,30 +77,51 @@ export const UnderlineNavLink = forwardRef(
     const textStyles: BetterSystemStyleObject = {
       whiteSpace: 'nowrap'
     }
+
+    const wrapperStyles = {
+      display: 'inline-flex',
+      paddingY: 1,
+      paddingX: 2
+    }
     const smallVariantLinkStyles = {
-      paddingY: 2,
+      paddingY: 1,
       fontSize: 0
     }
     const defaultVariantLinkStyles = {
-      paddingY: 3,
+      paddingY: 2,
       fontSize: 1
     }
 
     const linkStyles = {
       display: 'inline-flex',
       color: 'fg.default',
       textAlign: 'center',
-      borderBottom: '2px solid transparent',
```

Comment:
I had to remove the border entirely because of the focus style below. (The focus class on link element)

     &:focus': {
        outlineColor: 'accent.fg',
        borderRadius: 2,
        outlineOffset: '-6px',
        transition: '0.2s ease'
      }

The outline border can only be rounded with borderRadius. However when I added the borderRadius to the focus style, as this focus class gets triggered when **the item is selected as well**, it ended up rounding the orange border bottoms too. So I removed the borders from the links and add a standalone element and styled them to look like border bottom so that they won't affect from the focus styles. Please let me know what you think or if you have a different idea.

Reply:
with the new commit this is not relevant anymore